File: android-java/BackStack/app/src/main/java/com/example/backstack/MainActivity.java
package com.example.backstack;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;

import com.example.backstack.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {
    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        binding.goToBButton.setOnClickListener(view->
        {
            Intent pageB = new Intent(MainActivity.this,ActivityB.class);
            finish();
            startActivity(pageB);

        });

    }
}

File: android-kotlin/PlayOffs/app/src/main/java/com/example/playoffs/view/MainActivity.kt
package com.example.playoffs.view

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.example.playoffs.databinding.ActivityMainBinding
class MainActivity : AppCompatActivity() {
    private lateinit var binding : ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        var view = binding.root
        setContentView(view)



    }
    fun player2MatchClick(view: View){
        val intent = Intent(this, Player2MatchActivity::class.java)
        startActivity(intent)
    }
}